samples/rtl/rtl_tasklet_pipeline.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+# Copyright 2019-2020 ETH Zurich and the DaCe authors. All rights reserved.
+
+"""
+    Pipelined, axi-handshake compliant example that increments b from a up to 100.
+"""
+
+import dace
+
+import numpy as np
+
+# add symbol
+N = dace.symbol('N')
+
+# add sdfg
+sdfg = dace.SDFG('rtl_tasklet_pipeline')
+
+# add state
+state = sdfg.add_state()
+
+# define compile-time constant
+sdfg.specialize(dict(N=4))
+
+# add arrays
+sdfg.add_array('A', [N], dtype=dace.int32)
+sdfg.add_array('B', [N], dtype=dace.int32)
+
+# enable debugging output
+sdfg.add_constant("DEBUG", 1)
```

Comment:
I suggest to disable by default, and allow it to be set to true with a command line parameter